README.md

```diff
@@ -9,9 +9,9 @@ MyLocalTimes is developed with [react-native](https://facebook.github.io/react-n
 ---
 ## What do it do?
 
-This application is a method for consulting the timezone for all world and it utilized the [open source API](https://github.com/davidayalas/current-time) for obtain the timezone.
+This application is a method for consulting the timezone for all world and it utilizes the [open source API](https://github.com/davidayalas/current-time) to obtain the timezone.
 
-PS: in the future, this application could be upgraded on utilized [Moment.js library](https://momentjs.com/) for load the timezone from the local bundle, but to the moment for the correct utilized of the application is need the internet connection.
+PS: in the future, this application could be upgraded to utilize [Moment.js library](https://momentjs.com/) to load the timezone from the local bundle, but for the moment for the correct utilize of the application it is need an internet connection.
```

Comment:
Ops! This is my bad. It is no longer true, the App supports the Moment.js lib. Also, at the moment Moment.js is not the best choice.

Maybe we can add simply

> This application helps to consult the timezone for all world, and it uses Moment.js library to obtain the correct timezone.